public/app/plugins/panel/barchart/BarChartPanel.tsx

```diff
@@ -242,7 +242,17 @@ export const BarChartPanel: React.FunctionComponent<Props> = ({
         f.config.custom?.gradientMode === GraphGradientMode.Scheme &&
         f.config.color?.mode === FieldColorModeId.Thresholds;
 
-      return fromThresholds || f.config.mappings?.some((m) => m.options.result.color != null);
+      return (
+        fromThresholds ||
+        f.config.mappings?.some((m) => {
+          // ValueToText mappings have a different format, where all of them are grouped into an object keyed by value
+          if (m.type === 'value') {
+            // === MappingType.ValueToText
```

Comment:
this was on the previous line, until Prettier "fixed" it :roll_eyes: 

this is here because we currently have an issue with the schema generation which exports enums only as types, so the values cannot be used as RHS. @sdboyer is on it, but we can probably just land it like this, since that can take more time and we'll want this fix to land in the next patch release.

![image](https://user-images.githubusercontent.com/43234/206614051-9fc80a68-a898-409b-9173-5d77a068b4f1.png)

Reply:
fyi, this is already fixed in #55769 - i'm just loathe to pull it out and inevitably create a merge conflict there when it's already so close

i'll do it if that PR doesn't land by next week though